contactaccessrights/js/test/controllers/access-rights-ctrl_test.js

```diff
@@ -0,0 +1,33 @@
+define([
+  'common/angularMocks',
+  'access-rights/controllers/access-rights-ctrl'
+], function () {
+  'use strict';
+
+  describe('AccessRightsCtrl', function () {
+    var ctrl, modalSpy;
+
+    beforeEach(module('access-rights.controllers'));
+    beforeEach(inject(function (_$controller_, _$rootScope_) {
+      CRM = {
```

Comment:
You could use the same solution used in [appraisals](https://github.com/civicrm/civihr/blob/develop/uk.co.compucorp.civicrm.appraisals/js/test/globals.js) for these global vars